Composer/packages/client/src/Onboarding/TeachingBubbles/index.tsx

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import formatMessage from 'format-message';
+import { TeachingBubble } from 'office-ui-fabric-react/lib/TeachingBubble';
+
+import OnboardingContext from '../context';
+import { getTeachingBubble } from '../onboarding';
+import { StoreContext } from '../../store';
+
+import { teachingBubbleStyles, teachingBubbleTheme } from './styles';
+
+const TeachingBubbles = () => {
+  const [, forceRender] = useState();
+
+  const rerender = useCallback(() => forceRender({}), [forceRender]);
```

Comment:
this is a little tricky - let's comment here why this is necessary for future eyes on this code